R/account-find.R

```diff
@@ -17,7 +26,11 @@ findAccount <- function(accountName = NULL, server = NULL, error_call = caller_e
   }
 
   if (!is.null(accountName) && !is.null(server)) {
-    theseAccounts <- accounts[accounts$server == server & accounts$name == accountName, , drop = FALSE]
+    theseAccounts <- accounts[
+      accounts$server == server & accounts$name == accountName,
+      ,
+      drop = FALSE
+    ]
```

Comment:
This one looks weird, even though it's technically correct.

Reply:
I've rewritten to avoid the need to use a multiline expression.